src/dmd/dmangle.d

```diff
@@ -1267,6 +1267,14 @@ public:
         }
     }
 
+    override void visit(FuncExp e)
+    {
+        if (e.td)
+            mangleSymbol(e.td);
```

Comment:
Missing tests...

Reply:
Because this code only happens inside a struct literal the function type is resolved and `e.td` is always null.
What should I do? assert(0 or !e.td), comment out, just delete...